server/client.go

```diff
@@ -1349,6 +1349,11 @@ func (c *client) processConnect(arg []byte) error {
 	// Estimate RTT to start.
 	if c.kind == CLIENT {
 		c.rtt = computeRTT(c.start)
+
+		if c.srv != nil {
+			c.clearPingTimer()
+			c.srv.setFirstPingTimer(c, true)
```

Comment:
I don't think we need a boolean here if we call `c.setPingTimer()` in `createClient()`.